gcloud-java-storage/src/main/java/com/google/gcloud/storage/BucketInfo.java

```diff
@@ -434,33 +361,57 @@ Builder selfLink(String selfLink) {
       return this;
     }
 
+    /**
+     * Sets whether blob versioning should be enabled or not for the bucket.
+     */
     public Builder versioningEnabled(Boolean enable) {
       this.versioningEnabled = firstNonNull(enable, Data.<Boolean>nullOf(Boolean.class));
       return this;
     }
 
+    /**
+     * Sets the bucket's website index page.
```

Comment:
"Behaves as the bucket's directory index where missing objects are treated as potential directories."...
Similar to my suggestion in #356 I think we can improve on the comments of some of the setters/getters by using
a description similar to the ones described [here](https://cloud.google.com/storage/docs/json_api/v1/buckets#resource).